releasenotes/notes/0.16/dagcircuit-compose-fix-multiple-creg-7f3478401a874fbb.yaml

```diff
@@ -0,0 +1,10 @@
+---
+fixes:
+  - |
+    The :meth:`qiskit.circuit.QuantumCircuit.compose` and
+    :meth:`qiskit.dagcircuit.DAGCircuit.compose` methods had, in some cases,
+    incorrectly translated conditional gates if the input circuit contained
+    more than one :class:`~qiskit.circuit.ClassicalRegister`. This has been
+    fixed so the multiple :class:`~qiskit.circuit.ClassicalRegister` objects
+    are correcly handled.
+    Fixes `#4964 <https://github.com/Qiskit/qiskit-terra/issues/4964>`__
```

Comment:
Backported in 0.15.2.

Reply:
Removed in: https://github.com/Qiskit/qiskit-terra/pull/5142/commits/bdfce87f1dda045ec61dd740645aed055df041ee